subsys/net/l2/ethernet/ethernet.c

```diff
@@ -493,6 +493,17 @@ static void ethernet_update_tx_stats(struct net_if *iface, struct net_pkt *pkt)
 }
 #endif /* CONFIG_NET_STATISTICS_ETHERNET */
 
+static void ethernet_remove_l2_header(struct net_pkt *pkt)
+{
+	struct net_buf *buf;
+
+	/* Let's remove the buffer added in ethernet_fill_header() */
```

Comment:
Can the comment be more formal here? E.g. "First fragment is Ethernet header added in ethernet_fill_header()". (Btw, ethernet_fill_header() better be called ethernet_add_header(), ethernet_insert_header(), or ethernet_prepend_header())

Reply:
I don't see why we would need to change that function name. moreover in that PR.

Reply:
> I don't see why we would need to change that function name. moreover in that PR.

Well, at least I'd suggest to remove "Let's" from the comment, or it gives a feeling that we can either remove it or not, but ok, decided to remove it after all ;-).